composer.json

```diff
@@ -19,7 +19,8 @@
         "psr/http-message": "^1.0",
         "symfony/process": "^4.2",
         "thecodingmachine/safe": "^0.1.14",
-        "zendframework/zend-diactoros": "^2.1"
+        "zendframework/zend-diactoros": "^2.1",
+        "ext-tideways": "@stable"
```

Comment:
How do we make this work without `--ignore-platform-reqs`?

Reply:
Will also likely need to move the profiler initialisation to a file that isn't checked by static analysis, since profiler classes are not available

Reply:
Maybe I don't see it, but where do we need `ignore-platform-reqs`?

Reply:
Indeed, my snafu coming from https://support.tideways.com/article/90-install-on-heroku

Reply:
@Ocramius you are right, it should not be needed because of the config platform setting.

Reply:
Since the profiler variables are set conditionally: `class_exists('Tideways\Profiler')`, this shouldn't be here in `require`.

Reply:
Heroku needs these extensions set in `require` to enable them: https://devcenter.heroku.com/articles/php-support#using-optional-extensions.